### PR TITLE
added setting to skip x epochs when using  train_and_test_on_datasets

### PR DESCRIPTION
**Pull request by studentWUR**
## Summary:
For training it is not desirable to test at every epoch. This simple update solves this issue. By using a skip_epochs option in train_and_test_on_datasets

## Checklist:

* [ ] Your code is documented (To validate this, add your module to `tests/documentation_test.py`).
* [ ] Your code is tested with unit tests.
* [ ] You moved your Issue to the PR state.

### CODE REVIEW

File: baal/modelwrapper.py [hunk 1]
```diff
@@ -178,17 +179,21 @@ def train_and_test_on_datasets(
...
+                if te_loss < best_loss:
+                    best_epoch = e
+                    best_loss = te_loss
+                    if return_best_weights:
+                        best_weight = deepcopy(self.state_dict())
+
+                if patience is not None and (e - best_epoch) > patience and (e > min_epoch_for_es):
+                    # Early stopping
+                    break
+                hist.append(self.get_metrics())
```
Comment (Dref360) [inline baal/modelwrapper.py:194]: This is appending the metric twice I think. L184 already does it.

**Review by Dref360 — CHANGES_REQUESTED** [review]
Hello, thanks for the PR. 
We can merge it once the comment is addressed and the formatting is done.

To do so, you can run `make format`.

File: baal/modelwrapper.py (hunk 1 above)
Comment (studentWUR) [reply baal/modelwrapper.py:194]: Yes I made an mistake in the PR, sorry for that

Comment (parmidaatg) [reply baal/modelwrapper.py:194]: Hi @studentWUR, 
Thank you so much for your Pull Request. Would you mind pushing the requested change so that we can merge this branch?

Comment (studentWUR) [reply baal/modelwrapper.py:194]: Sorry for my late reply; pushing a pull request is new for me; so I am slightly confused. In my forked repo I fixed the abovementioned typo, but how to push this to your git? Can you merge it from mine repo

**Review by Dref360 — APPROVED** [review]
LGTM!
